hr_attendance_management/models/hr_employee.py

```diff
@@ -46,6 +59,11 @@ class HrEmployee(models.Model):
     #                             FIELDS METHODS                             #
     ##########################################################################
 
+    @api.multi
+    def _compute_annual_balance_date(self):
+        self.env['base.config.settings'].create({})\
```

Comment:
compute method should assign the result to the field.
```suggestion
        for employee in self:
            employee.annual_balance_date =\
                self.env['base.config.settings'].create({})\
```